developers/wcs/platform/users-and-organizations.mdx

```diff
@@ -0,0 +1,131 @@
+
+## Edit organization settings
+
+User editable configuration settings are on the organization settings tab.
+
+To open the settings tab, click `Organization settings` at the bottom of the organizations pull down menu.
+
+<img src={OrgSettings} width="50%" alt="Organization settings button"/>
+
+### Create an organization
+
+To create a new organization, follow these steps:
+
+1. Click the organization pulldown menu in the WCD console.
+
+    <img src={OrgPulldown} width="50%" alt="Organization pulldown menu"/>
+
+2. Click "Add new organization".
+1. Enter the organization name.
+
+WCD automatically switches the console view to the new organization.
+
+### Switch organizations
+
+To switch between organizations, select an organization name in the organization pulldown menu in the WCD console.
+
+### Edit an organization name
+
+To edit an organization name, follow these steps:
+
+1. Open the [organization settings](#edit-organization-settings) panel.
+1. Click the `Update` button.
+
+<img src={OrgUpdateName} width="75%" alt="Update name button"/>
+
+3. Edit the name.
+1. Click the check mark to accept the change.
+
+<img src={OrgUpdateCheckMark} width="75%" alt="Update name button"/>
+
+### Delete an organization
+
+To delete an organization, [contact support](mailto:support@weaviate.io).
+
+## Manage users
+
+Any member of an organization can add or remove a user. A new user has to have a WCD account.
+
+### Add a user
+
+A WCD user can be a member of more than one organization. Once a user creates an account in WCD, you can add that user to your organization.
+
+:::note Cluster restart
+<RestartTheCluster/>
+:::
+
+To add a user, follow these steps:
+
+1. Ask the user for WCD account details.
+
+   - If the user has an account, note the email address.
+   - If the user does not have an account, [create a new account](/developers/wcs/create-account).
+
+2. Open the [organization settings](#edit-organization-settings) panel.
+1. Enter the user's email address in the `Users` section. Click `Add user`.
+
+WCD sends an email to the user when the user is added to the organization.
+
+### Remove a user
+
+To remove a user, follow these steps:
+
+:::note Cluster restart
+<RestartTheCluster/>
+:::
+
+1. Open the [organization settings](#edit-organization-settings) panel.
+1. Find the user's email address in the `Users` section.
+1. Click the trashcan icon to delete the user.
+1. Click `Confirm`.
+
+If you have clusters that are created by the deleted user, those clusters are still available after the restart.
+
+## Billing
+
+[Billing](/developers/wcs/platform/billing.mdx) is done on an organization level.
+
+Once a billing account is configured, every user in an organization can create new clusters or modify existing ones.
+
+Each cycle the account gets a bill that includes charges for all of the clusters. The charges are itemized by cluster, not by user.
```

Comment:
The account will get a stand alone invoice for each cluster, on a 30 day cycle from when each cluster was created